src/w2grid.js

```diff
@@ -474,7 +474,7 @@
                     }
                 }
                 if (match && returnIndex !== true) recs.push(this.records[i].recid);
-                if (match && returnIndex === true) recs.push(i);
+                if (match && returnIndex === true) recs.push(this.records[i]);
```

Comment:
Then documentation is wrong:
http://w2ui.com/web/docs/1.5/w2grid.find
It says "boolean, if true returns records not their recids"
And also the way it works as described in documentation is much more useful (to get whole record object than just array index which means and does nothing).